pkg/cluster/pod.go

```diff
@@ -45,6 +47,59 @@ func (c *Cluster) getRolePods(role PostgresRole) ([]v1.Pod, error) {
 	return pods.Items, nil
 }
 
+// markRollingUpdateFlagForPod sets the indicator for the rolling update requirement
+// in the Pod annotation.
+func (c *Cluster) markRollingUpdateFlagForPod(pod *v1.Pod, msg string) error {
```

Comment:
I would assume `set` implies I can set it either `true` or `false` (otoh, in that case it could also be called `enableRollingUpdateFlagForPod` then). It was a suggestion by @Jan-M